explanations/machine-learning-terms.md

```diff
@@ -41,6 +41,14 @@ Researchers often invent complicated names for simple things. This page will hel
     <p>Technically, it's a statistical procedure that identifies and sorts a small number of uncorrelated variables, called principal components, in such a way that the first one has the most scattered data. So if we want to reduce from <tt>n</tt> to <tt>k</tt> dimensions, we keep the <tt>k</tt>'s first principal components. But the lower is <tt>k</tt> the greater is the information loss.</p>
 </td>
 </tr>
+<tr>
+<td valign="top">Kernel</td>
+<td>
+    <p>A function that helps to separate datasets that can't be separated with a classic approach.</p>
```

Comment:
I have the feeling the word "linear" is missing here... Maybe "...with a classic linear approach." ?

Reply:
Hehe, I tried to simplify as much as possible, but I may have oversimplified a bit... you are right indeed, I need to add the notion of linearity. Your suggestion is good, I will use it.